scripts/os_configure/install_software_20.bash

```diff
@@ -156,11 +180,6 @@ then
   sudo apt update
   sudo apt -y install gh
```

Comment:
should be moved up to be installed in all cases. Very usefuly tool

Reply:
Done